.github/CODEOWNERS

```
@@ -1,6 +1,6 @@
 *       @Jigsaw-Code/outline-dev
 
-/cordova-plugin-outline/  @Jigsaw-Code/outline-networking-owners
+/src/cordova/plugin/  @Jigsaw-Code/outline-networking-owners
 /third_party/             @Jigsaw-Code/outline-networking-owners
```

I'm not sure whether it is because of the GitHub display issue, they seems not aligned.

```suggestion
/third_party/         @Jigsaw-Code/outline-networking-owners
```